docs/sql-migration-guide.md

```diff
@@ -24,6 +24,8 @@ license: |
 
 ## Upgrading from Spark SQL 3.1 to 3.2
 
+  - In Spark 3.2, PostgreSQL JDBC dialect uses StringType for MONEY and MONEY[] is not supported due to the JDBC driver for PostgreSQL can't handle those types properly. In Spark 3.1 or earlier, DoubleType and ArrayType of DoubleType are used respectively.
```

Comment:
I have the same comment here with https://github.com/apache/spark/pull/31491#discussion_r573709268

Reply:
I've replied at https://github.com/apache/spark/pull/31491#discussion_r573846295 .

Reply:
@maropu Any more concern?